src/features/defi/providers/idle/components/IdleManager/Deposit/IdleDeposit.tsx

```diff
@@ -74,6 +74,7 @@ export const IdleDeposit: React.FC<IdleDepositProps> = ({
   useEffect(() => {
     ;(async () => {
       try {
+        if (state.userAddress && state.opportunity) return
```

Comment:
Note l.81 will go away with multi account feature, all we need is an `AccountId` / `AccountNumber`, and this is not the place to get/derive it